cmd/flowserver/flow.go

```diff
@@ -37,9 +32,9 @@ func (r *flowResponse) MarshalJSON() ([]byte, error) {
 
 type startRequest struct {
 	Environment *json.RawMessage       `json:"environment"`
-	Flows       json.RawMessage        `json:"flows"    validate:"required"`
-	Channels    []json.RawMessage      `json:"channels,omit_empty"`
-	Contact     json.RawMessage        `json:"contact"  validate:"required"`
+	Assets      json.RawMessage        `json:"assets"           validate:"required"`
+	Flow        flows.FlowUUID         `json:"flow_uuid"        validate:"required"`
```

Comment:
I was saying we now have the possible error of that flow uuid not being present in the flow list, which isn't possible when you just say the first flow will be started.

That said, it does feel weird to have that behavior of implicitly starting the first flow in `assets` so I agree this is the right change.